assets/app/view/game/stock_market.rb

```diff
@@ -150,6 +151,18 @@ def token_props(corporation, index = nil, num = nil, spacing = nil)
           props[:attrs][:title] = "#{corporation.name} has operated"
           props[:style][:opacity] = '0.6'
           props[:style][:clipPath] = 'polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%)'
+          border = 'dashed'
```

Comment:
remember to include Lib::Color

Reply:
otherwise LGTM

Reply:
> remember to include Lib::Color

What for? player_colors is in Lib::Settings, which is already included.